www/js/services.js

```diff
@@ -45,15 +45,13 @@ angular.module('pvta.services', ['ngResource'])
 
 .factory('StopList', function(){
   var stopsList = {};
-  var pushToList = function(stop){
-    var id = stop.StopId
-    stopsList[id] = stop;
-  };
+
   var pushEntireList = function(list){
-    for(var i = 0; i < list.length; i++){
-      var id = list[i].StopId;
-      stopsList[id] = list[i];
-    }
+    list = _.uniq(list, 'Name');
+    list = _.sortBy(list, 'Name');
```

Comment:
Yep. I even read that line in the docs and didn't think of it. Good catch, thanks!